README.md

```diff
@@ -26,15 +26,23 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
   ```
 - Build the binary file.
   ```
-  go build -i -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
+  go build -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
   ```
-- Run create the project command.
+- Create the project from a template on a **main** branch.
   ```
```

Comment:
Can we add an empty line before it?

Reply:
Sure. Fixed in e6e50849fc249d06e49250271c2d33233b3f1001